client/src/ui/atoms/form/expanding-textarea.tsx

```diff
@@ -10,8 +10,8 @@ export default function ExpandingTextarea(
 
   const resizeCallback = useCallback(
     (node: HTMLTextAreaElement) => {
-      if (value && node && node.scrollHeight + 2 > node.clientHeight) {
-        node.style.height = `${node.scrollHeight + 2}px`;
+      if (value && node && node.scrollHeight > node.clientHeight) {
+        node.style.height = `${node.scrollHeight}px`;
```

Comment:
This bit is wrong: the css `height` property includes the border, so without accounting for it we still have a scrollbar after expanding.

Ideally we'd get the actual border heights from the element to add to `scrollHeight` here.

Reply:
Yes, but adding 2px is fine for now.